list_koji_tags.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+from ruamel.yaml import YAML
```

Comment:
If you're not writing YAML, I'd use vanilla yaml
```suggestion
import yaml
```